tests/CMakeLists.txt

```diff
@@ -13,17 +13,30 @@ file(
     DESTINATION
     ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} )
 
+file(
+    COPY
+    ganon-classify/data/bacteria_group_bin.txt
+    ganon-classify/data/classify.filter
+    ganon-classify/data/classify_output.txt
+    ganon-classify/data/simulated.1.fq
+    ganon-classify/data/simulated.2.fq
+    DESTINATION
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} )
+
 add_executable( ganon-tests
     aux/Aux.hpp
     setup/Catch2.setup.cpp
-    ganon-build/GanonBuild.test.cpp
-    utils/SafeQueue.test.cpp )
+    #ganon-build/GanonBuild.test.cpp
```

Comment:
@benvenutti do you have any idea why it breaks when compiling them together? it works separately

Reply:
Ok, I've found the problem. It's a class of symbols. Funny thing is, instead of getting a compiler error, the build succeeds but the binary get the Config classes mixed up. You can merge this and I will fix it on a new branch.